Makefile.toml

```diff
@@ -10,7 +10,7 @@ reduce_output = true
 default_to_workspace = false
 
 [env]
-DEV_FEATURES={ value = "storage-mem,http,scripting", condition = { env_not_set = ["DEV_FEATURES"] } }
+DEV_FEATURES={ value = "storage-mem,scripting,http,ml", condition = { env_not_set = ["DEV_FEATURES"] } }
```

Comment:
do we want to enable ml by default when we run or build the server locally?

Reply:
yes the build compiles the `ONNX` runtime into the rust binary and then links to it on starting the server. This means that the user does not have to worry about clashes with other machine-learning runtimes. The only time this will not work is with the `WASM` builds